src/driver/franka_plan_runner.cc

```diff
@@ -738,6 +760,61 @@ void FrankaPlanRunner::IncreaseFrankaTimeBasedOnStatus(
   }
 }
 
+bool FrankaPlanRunner::IsStartFarFromCurrentJointPosition() {
+  // Maximum change in joint angle between two confs
+  auto max_ang_distance =
+      utils::max_angular_distance(start_conf_franka_, start_conf_plan_);
+  if (max_ang_distance > params_.kMediumJointDistance) {
+    // far from start conf. return true
+    dexai::log()->error(
+        "JointPositionCallback: Discarding plan, mismatched start position."
+        " Max distance: {} > {}",
+        max_ang_distance, params_.kMediumJointDistance);
+    return true;
+  } else if (max_ang_distance > params_.kTightJointDistance) {
+    dexai::log()->warn(
+        "JointPositionCallback: max angular distance between franka and "
+        "start of plan is larger than 'kTightJointDistance': {} > {}",
+        max_ang_distance, params_.kTightJointDistance);
+  }
+  // not far from start conf. return false
+  return false;
```

Comment:
final return outside else but else is used after return, need refactor

Reply:
this pattern is undesirable, because your `else` does nothing. the recommended structure is
```
if ...
  return
if ...
  dexai::log()->warn(..
return false;
```

Reply:
removed `else`